receiver/vcenterreceiver/metadata.yaml

```diff
@@ -568,6 +575,14 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.vm.cpu.time:
+    enabled: false
+    description:  CPU time spent in idle, ready or wait state.
+    unit: "%"
+    gauge:
+      value_type: double 
+    attributes: [cpu_state, object_name]
+    extended_documentation: As measured over the most recent 20s interval.
```

Comment:
I'm not familiar with this component, do you have a reference for where this "20s interval" is coming from?

Reply:
This is hardcoded in our client. @StefanKurek suggested this and you can find about this in our discussion [here.](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37489#issuecomment-2622765873)

Reply:
Just reiterate the previous point, you can see 20s hard coded [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/vcenterreceiver/scraper.go#L280)